src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -1,14 +1,17 @@
 <%@ tag description="instructorFeedbacks and instructorHome - Feedback Session publish/unpublish button" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
+<%@ attribute name="showButtonAndTooltip" type="java.lang.Boolean" required="true" %>
 <%@ attribute name="buttonType" required="true" %>
-<a class="btn ${buttonType} btn-tm-actions session-${publishButton.actionNameLowercase}-for-test"
-   href="${publishButton.actionLink}"
-   title="${publishButton.tooltipText}"
-   data-toggle="tooltip"
-   data-placement="top"
-   data-fsname="${publishButton.feedbackSessionName}"
+<a href="${publishButton.actionLink}" 
+   class="<c:if test="${showButtonAndTooltip}">${buttonType}</c:if> session-${publishButton.actionNameLowercase}-for-test"
```

Comment:
@leeyimin If you are talking about extra space next to `Results` drop-down button, then that is because of the `padding-right-25px` as seen in this image
![screenshot from 2017-05-23 19-17-24](https://cloud.githubusercontent.com/assets/9383897/26357456/e6fdba00-3fec-11e7-9c54-798342c5ce85.png)